code/web/src/modules/survey/api/actions.js

```diff
@@ -81,6 +82,16 @@ export const clearSurvey = () => {
   }
 }
 
+//retake survey
+export const retakeSurvey = () => {
+  return dispatch => {
+    dispatch({
+      type: 'RETAKE_SURVEY',
+      page: 1
+    })
+  }
+}
+
```

Comment:
This is going to return you to the first page but does it undo your selections? As a user when I see "Retake Survey" I think of starting over from scratch.

Reply:
It does not reset selections right now. This could be done if passed the existing products object that is in store, ran it through a 'for in' loop that would mark all products.selected to 'false'. I wasn't sure if we wanted it to be reset or changed. Given the name of the button, I guess resetting all the selections would make more sense.  I have fine with it how it is, though.  We could even just change the name of the button to 'Review Survey'.

Reply:
I think that name change would clarify the UX. I can think of a couple other relatively easy solutions but that is definitely the easiest.